public/sw.js

```diff
@@ -0,0 +1,51 @@
+const CACHE_VERSION = 'v1'
+const ROOT_PATH = 'https://passcryptum.com/'
+
+addEventListener('install', () => {
+  skipWaiting()
+})
+
+addEventListener('activate', event => {
+  clients.claim()
+
+  event.waitUntil(
+    caches.keys().then(keys =>
+      keys.forEach(key => {
+        caches.delete(key)
```

Comment:
IDE/линтер могут ругаться на необработанные промисы. Чтобы избавиться от этого, необходимо добавить после возвращённого промиса пустой обработчик.

```suggestion
  skipWaiting().then()
})

addEventListener('activate', event => {
  clients.claim().then()

  event.waitUntil(
    caches.keys().then(keys =>
      keys.forEach(key => {
        caches.delete(key).then()
```